sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/QueryComparisonOperator.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.DigitalTwins.Core.QueryBuilder
+{
+    /// <summary>
+    /// <see href="https://docs.microsoft.com/en-us/azure/digital-twins/reference-query-operators#comparison-operators">Comparison operators</see> in the ADT Query language.
+    /// </summary>
+    public enum QueryComparisonOperator
+    {
+        /// <summary>
+        /// Equality operator.
+        /// </summary>
+        Equal = 1,
```

Comment:
Why not Equals, NotEquals?

Reply:
GreaterThanOrEquals, LessThanOrEquals

Reply:
I thought singular words instead of plurals read better in the query. For instance:

`Where().Comparison("Temperature", QueryComparisonOperator.Equal, 50)` reads a little better to me since in my head I'm going "Temperature is equal to 50". I can totally see how someone would find Equals more readable through. That can be something we ask about in our feedback session.

Reply:
This isn't singular or plural. Equals is a verb with a specific tense. 5 equals 2+3 isn't plural.

Reply:
That's true. I still think (imho) it's more readable this way and we should ask a broader audience their thoughts on which option that they prefer.